zippkg/helpers.go

```diff
@@ -145,7 +146,7 @@ func AddFileFromBytes(z *zip.Writer, zipPath string, data []byte) error {
 // ExtractToDiskTmp extracts a zip file to a temporary file in a given path,
 // returning the name of the file.
 func ExtractToDiskTmp(f *zip.File, path string) (string, error) {
-	tmpFile, err := ioutil.TempFile(path, "zz")
+	tmpFile, err := tempstorage.TempFile(path, "zz")
```

Comment:
need to make sure files are closed?

Reply:
there is defer tmpFile.Close() below, on line 154